src/Attributes/ValidateEntityLoad.php

```diff
@@ -8,7 +8,9 @@
 use Consolidation\AnnotatedCommand\CommandData;
 use Consolidation\AnnotatedCommand\CommandError;
 use Drush\Utils\StringUtils;
+use JetBrains\PhpStorm\Deprecated;
 
+#[Deprecated(replacement: 'Copy \Drush\Commands\core\ImageFlushCommand::validateEntityLoad into command and call during execute()')]
```

Comment:
Could we maybe use a [command hook](https://symfony.com/doc/current/components/console/events.html#the-consoleevents-command-event) to find and dispatch validate hooks?

Reply:
I have experimented with that approach earlier in this PR and can't decide whats best. Currently the validation is done right in execute() and isn't not DRY. See [ImageFlushCommand](https://github.com/drush-ops/drush/pull/6135/files#diff-3b7cd99de69343fefe9c81829f97ac88a82904e64b45cdda528de7974483518aR65-R101). Compare with https://github.com/drush-ops/drush/blob/f923a801dd50d4aa17fd2390790336bc962785a3/src/Listeners/ValidateModulesEnabledListener.php and its associated Attribute. IMO the current approach is less magical and I'm slightly valuing that these days.